packages/selection/README.md

```diff
@@ -29,20 +29,20 @@ import { useSelection } from '@zendeskgarden/container-selection';
 const items = ['Item 1', 'Item 2', 'Item 3'];
 
 const Selection = ({ direction }) => {
-  const { focusedItem, selectedItem, getContainerProps, getItemProps } = useSelection({
+  const { focusedValue, selectedValue, getGroupProps, getElementProps } = useSelection({
```

Comment:
Looks like the examples need to be updated with the required `values` parameter.